ech/README.md

```diff
@@ -0,0 +1,43 @@
+# ECH Example that connects to a cloudflare test server which supports ech
+
+This test is successful if the cloudflare http response shows that `sni=encrypted`.
```

Comment:
Please include steps for building wolfSSL to include the ECH and HPKE features...

`./configure --enable-hpke --enable-ech && make && sudo make install`